src/scripts.js

```diff
@@ -46,26 +79,68 @@ function displayAllRecipes() {
 }
 
 function viewRecipeDetail(event) {
-    //GOAL: incoporate instructions and total cost into recipe details page
+  viewRecipeInstructions(event);
+  viewRecipeTotalCost(event);
+  viewRecipeIngredients(event);
+}
+
+function viewRecipeIngredients(event) {
+  const foundRecipe = recipeRepository.recipes.find((current) => {
+      return current.id === findId(event);
+  })
```

Comment:
This is not a high priority, but lines 88-90 are the same as 138-140 and 117-119. I'm wondering if we can move this to line 81 to just have it once in the `function viewRecipeDetail(event)`